assignments/assignment-8/assignment-8.4.html

```diff
@@ -0,0 +1,69 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+        <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+        <title>Assignment-8.4</title>
+    </head>
+    <body>
+        <h1>Customer Feedback Survey</h1>
+        <form>
+            <label for="fullNameInput">Full Name</label>
+            <input type="text" name="fullName" id="fullNameInput" />
+            <div>
+                <h2>Rate your Overall Experience</h2>
+                <label for="excellentSelect">Excellent</label>
+                <input
+                    type="radio"
+                    name="experienceRating"
+                    id="excellentSelect"
+                />
+                <label for="goodSelect">Good</label>
+                <input type="radio" name="experienceRating" id="goodSelect" />
+                <label for="averageSelect">Average</label>
+                <input
+                    type="radio"
+                    name="experienceRating"
+                    id="averageSelect"
+                />
+                <label for="poorSelect">Poor</label>
+                <input type="radio" name="experienceRating" id="poorSelect" />
```

Comment:
I think you should choose radio input not select, because radio is choosing on type